utils/common.js

```diff
@@ -326,7 +333,7 @@ export const formattedNormalize = (number, lang = 'np') => {
 };
 
 export const leftPad = (number, length, pad = '0') => {
-    if (number === '' || number === undefined) {
+    if (isFalsy(number, [''])) {
```

Comment:
Better to use `isNotDefined` for number

Reply:
It shouldn't matter here, isFalsy is different from isNotDefined for boolean values